### DISCUSSION TITLE
Keyboard

### DISCUSSION THREAD

**Discussion by timboh72** [guysoft/FullPageOS#529] [Q&A]
Good morning, I have found FullPageOS through the Pi imager tool and it will be good to allow someone with disability to use without the ability to access the file system and be locked into one or two pages only and therefore not delete anything.
Whilst this is probably a Linux question I would appreciated if there is any way to reenable the keyboard and mouse through the config.txt or similar so the user can still interact with the browser.
My Linux knowledge is limited and I have a way to go to setup a hardened PiOS with limited access.
Thank you in advance

**Comment by guysoft** [comment]
Keyboard and mouse are enabled in FullPageOS by default. You can click on links and type. What are you tying to do where they don't work?

**Comment by LennartSea** [comment]
I have a similar problem. Mouse and keyboard works but where to xonfigure keyboard type ? 
I need Nordic type.

**Comment by guysoft** [comment]
@LennartSea 
rpi-config, it Rpi OS based:
https://linuxconfig.org/how-to-change-keyboard-layout-on-raspberry-pi

I am turning this to a discussion

**Reply by Mattat01** [reply]
@guysoft,  thanks for this, it is just what I was after (previous efforts treated screen touch as moving the mouse instead of scrolling, but I appreciate that you have that working as expected by default). On the question of keyboards though, without access to the GUI settings I cannot work out how to set the pi to display the virtual keyboard by default. Through the GUI there is the option ot show the keyboard never, always or when a KB is not attached. Do you know how to set it to always via ssh?

Thanks
